clippy_lints/src/unicode.rs

```diff
@@ -112,6 +117,15 @@ fn check_str(cx: &LateContext<'_, '_>, span: Span, id: HirId) {
                     escape(string.nfc())
                 }
```

Comment:
With the `span_lint_and_sugg` you can/should remove the format (inclusive the colon). The message should also start lowercase.

Reply:
thanks for the hints and suggestions :slightly_smiling_face: